runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionFormPanel/widgets/index.tsx

```diff
@@ -12,7 +12,7 @@ const TypographyWidget = dynamic(() => import('./TypographyWidget'), { ssr: fals
 const DatePickerWidget = dynamic(() => import('./DatePickerWidget'), { ssr: false });
 
 const customWidgets = {
-    TextWidget: ElementAwareAutocompleteWidget,
+    TextWidget: CustomTextWidget,
```

Comment:
so maybe just name `CustomTextWidget` as `TextWidget` then?

Reply:
rjsf lib already has TextWidget which I override here. Are you sure this change  won't be confusing in the future?